src/components/beregningOgSimulering/beregning/Beregning.tsx

```diff
@@ -77,7 +79,8 @@ function getInitialValues(fradrag: Fradrag[], begrunnelse?: Nullable<string>): F
     };
 }
 
-const Beregning = (props: VilkårsvurderingBaseProps) => {
+type Søker = { søker: personApi.Person };
+const Beregning = (props: VilkårsvurderingBaseProps & Søker) => {
```

Comment:
```suggestion
const Beregning = (props: VilkårsvurderingBaseProps & { søker: Person }) => {
```
Ville bare importert Person direkte, og da kan man like så godt slå de sammen siden typen ikke brukes noe sted ellers. Fliskespikkeri dog 🤷‍♂️

Reply:
Er enig, blev sån bara fordi det blev gjord sån på andre steder. Men endrer til det ovan.